tox.ini

```diff
@@ -80,9 +80,9 @@ commands_pre =
     # the rest is for good measure
     headless: pip uninstall -y pytest-qt qtpy pyqt5 pyside2
 commands =
-    !headless: python -m pytest {env:PYTEST_PATH:} --color=yes --basetemp={envtmpdir} \
+    !headless: python -m pytest . --color=yes --basetemp={envtmpdir} \
         --cov-report=xml --cov={env:PYTEST_PATH:napari} --ignore tools --maxfail=5 \
-        --json-report --json-report-file={toxinidir}/report-{envname}.json {posargs}
+        --json-report --json-report-file={toxinidir}/report-{envname}.json {posargs} \
```

Comment:
Do you need to try to fix this file in this PR? Seems like that's causing the check failures.